bin/webhook_server

```diff
@@ -15,8 +15,11 @@ import logging
 import BaseHTTPServer
 from docopt import docopt
 import json
+import hmac
+import hashlib
+import os
```

Comment:
best practice for imports:
1. first comes standard library imports, in alphabetical order
2. 3rd party libraries, in alphabetical order
3. project imports, in alphabetical order.

Reply:
Done